src/CodeGenerator/Templates/Resources/TemplateContent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Reflection;
+using Pedantic.IO;
+
+namespace CodeGenerator.Templates.Resources
+{
+    internal static class TemplateContent
+    {
+        private static Assembly Assembly => typeof(TemplateContent).Assembly;
+        private static string Namespace => typeof(TemplateContent).Namespace;
+
+        internal static string Class => EmbeddedResource.ReadAllText(Assembly, $"{Namespace}.Class.txt");
```

Comment:
I should drop `EmbeddedResource` and write my own code for this - given the code is simple and the library is in prerelease and seems to be abandoned